drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -318,8 +320,9 @@ protected void replaceColumn(int columnIndex, GridColumn<?> column) {
         String columnId = ((ScenarioGridColumn) column).getInformationHeaderMetaData().getColumnId();
         ExpressionIdentifier ei = ExpressionIdentifier.create(columnId, FactMappingType.valueOf(group));
         commonAddColumn(columnIndex, column, ei);
+        /* Restoring the expected columns dimension, overriding the automatic resizing */
```

Comment:
@yesamer 
I personally do not have nothing against such comments, but keep in mind not everyone agrees on that.

Reply:
I'll keep it in mind.